packages/@uppy/core/src/index.js

```diff
@@ -1032,8 +1032,6 @@ class Uppy {
         bytesUploaded: data.bytesUploaded,
         bytesTotal: data.bytesTotal,
         percentage: canHavePercentage
-          // TODO(goto-bus-stop) flooring this should probably be the choice of the UI?
-          // we get more accurate calculations if we don't round this at all.
           ? Math.round((data.bytesUploaded / data.bytesTotal) * 100)
```

Comment:
I think the idea is when flooring, you won't get to `100%` until the upload is actually done. Currently, you get there as soon a 99.5% of the bytes have been sent.

Reply:
Discussed with @goto-bus-stop, we seem to be fine with the inaccuracy.